docs/book/src/developer/providers/migrations/v1.10-to-v1.11.md

```diff
@@ -557,19 +572,22 @@ If you are at this stage, you must use following util packages from CAPI v1.11 (
 ```go
 import (
 	...
-    conditions "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/conditions"
+    v1beta1conditions "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/conditions"
     v1beta2conditions "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/conditions/v1beta2"
```

Comment:
In CAPV we aliased them the other way around:
```suggestion
    deprecatedconditions "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/conditions"
    deprecatedv1beta2conditions "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/conditions/v1beta2"
```
So we e.g. don't messup:
* this `v1beta1conditions "sigs.k8s.io/cluster-api/util/deprecated/v1beta1/conditions"`
* vs `v1beta1conditions "sigs.k8s.io/cluster-api/util/conditions/v1beta1"`